pkg/cli/job/run.go

```diff
@@ -72,23 +73,14 @@ func RunJob() error {
 		return err
 	}
 
-	req, err := populateResourceListV1(launchJobFlags.Requests)
```

Comment:
why remove this?

Reply:
this function runs twice to get arg 'req' and 'limit', and the two args are used only when the default job 'test' is started.

Reply:
ok, got that.